TIL/topics/java_basic/02.클래스.md

```
@@ -87,4 +87,83 @@ Kinoko kinoko = new Kinoko();
+| **HEAP** | 동적으로 할당되는 메모리 영역 | `malloc()`, `new` 등 | 실행 중 동적 할당 |
+| **STACK** | 지역변수 및 함수 호출 정보 저장 | 지역 변수, 매개변수 등 | 실행 중 자동 할당 |
+
+
+<hr>
+
+
+### 👻 기본형(privitive) vs 참조형(reference)
+
+- 기본형 `primitive` : int, double, float, bool 등 값을 저장하기 위한 기본형 타입
+- 참조형 `reference` : new로 생성되는 인스턴스인 레퍼런스 타입
+    - 초기값이 `null` 로 설정된다
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)